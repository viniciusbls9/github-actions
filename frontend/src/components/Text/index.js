import styled from 'styled-components';

const StyledText = styled.h1`
    color: orange;
`;


export default function Text({children, ...props}) {
    return (
        <StyledText props={{...props}}>
            {children}
        </StyledText>
    )
}