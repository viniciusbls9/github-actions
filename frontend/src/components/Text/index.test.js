import Text from './index';
import { render } from '../../infra/test/testUtils';

describe('<Text />', () => {
    it('displays the component', () => {
        const { container } = render(<Text>Text</Text>);
        expect(container).toMatchSnapshot();
    });
});
