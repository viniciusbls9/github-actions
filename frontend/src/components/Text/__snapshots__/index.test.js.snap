// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Text /> displays the component 1`] = `
.c0 {
  color: orange;
}

<div>
  <h1
    class="c0"
  >
    Text
  </h1>
</div>
`;
